android/assets/jsons/translations/French.properties

```diff
@@ -693,9 +693,9 @@ The [questName] quest for [civName] has ended. It was won by [civNames]. =
 The resistance in [cityName] has ended! = la résistance dans [cityName] est finie!
 [cityName] demands [resource]! = [cityName] demande [resource]!
  # Requires translation!
-Because they have [resource], the citizens of [cityName] are celebrating We Love The King Day! = 
+Because they have [resource], the citizens of [cityName] are celebrating We Love The King Day! = Parce qu'ils ont [resource], les citoyens de [cityName] célèbrent la Fête du Roi !
  # Requires translation!
-We Love The King Day in [cityName] has ended. = 
+We Love The King Day in [cityName] has ended. = La Fête du Roi est terminée à [cityName].
```

Comment:
Majority of the other translations here uses a lower case 'roi' - consistency is good